docs/modules/rust-guide/pages/rust-counter.adoc

```diff
@@ -31,18 +31,36 @@ Like the other sample programs, this tutorial demonstrates a simple, but realist
 * Deploy the canister on the local {IC} network.
 * Invoke the canister methods to increment then read the value of a counter.
 
+[[before-you-begin]]
 == Before you begin
 
-Before starting the tutorial, verify the following:
+Before you start your project, verify the following:
+
+* You have an internet connection and access to a shell terminal on your local macOS or Linux computer.
 
 * You have downloaded and installed the Rust programming language and Cargo as described in the link:https://doc.rust-lang.org/book/ch01-01-installation.html[Rust installation instructions] for your operating system.
 
-* You have downloaded and installed the {sdk-short-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
++
+[source,bash]
+----
+curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+----
+
+* You have downloaded and installed the {sdk-long-name} package as described in link:../../quickstart/quickstart{outfilesuffix}#download-and-install[Download and install].
+
+* You have cmake installed. For example, use Homebrew with the following command:
+
++
+[source,bash]
+----
+brew install cmake
+----
+For instructions on how to install Homebrew, see the link:https://docs.brew.sh/Installation[Homebrew Documentation]
 
-* You have downloaded the latest version of the {cdk-long-name} from the link:https://crates.io/[Rust community’s crate registry] or from the link:https://github.com/dfinity/cdk-rs[Rust CDK] repository.
+* You have stopped any {IC} network processes running on the local computer.
 
-* You have stopped any {IC} network processes running on the local
-computer.
+If you aren’t sure how to open a new terminal shell on your local computer, run commands in a terminal, or how to check for and install packages, see link:newcomers{outfilesuffix}[Preliminary steps for newcomers]. 
```

Comment:
This link won't work unless you've added this to this repo and docs structure. Need the link to point to the topic in the quickstart module:
```suggestion
If you aren’t sure how to open a new terminal shell on your local computer, run commands in a terminal, or how to check for and install packages, see link:../quickstart/newcomers{outfilesuffix}[Preliminary steps for newcomers]. 
```